charts/kubeflow-trainer/templates/manager/deployment.yaml

```diff
@@ -30,7 +30,12 @@ spec:
     metadata:
       labels:
         {{- include "trainer.manager.selectorLabels" . | nindent 8 }}
+      annotations:
+        traffic.sidecar.istio.io/excludeInboundPorts: "9443"
     spec:
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
I actually fixed it upstream in istio a few days ago and we can now drop it altogether :-)

Reply:
For older installations it might help and for newer there is now harm